### PR TITLE
Lab5

### PR DESCRIPTION
**Pull request by danmory**
@bkmceh @ezio-42

### CODE REVIEW

**Comment by ezio-42** [comment]
LGTM

File: ansible/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Ansible Info
+
+## Prerequisites
+
+1. Install the following:
+
+   1. [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/index.html)
```
Comment (bkmceh) [inline ansible/README.md:7]: So its like 1.1 and 1.2? But anyway, nice work, im proud of u

File: ansible/plugins/inventory/yacloud_compute.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        super(InventoryModule, self).parse(inventory, loader, path)
+
+        self._read_config_data(path)
+        self._init_client()
+
+        self._get_clouds()
+        self._get_folders()
+
+        self._get_all_hosts()
+        self._process_hosts()
```
Comment (bkmceh) [inline ansible/plugins/inventory/yacloud_compute.py:143]: like this huge python code, good job

**Comment by bkmceh** [comment]
Actually good job, what I can add more idk

**Pull request closed without merging**